### DISCUSSION TITLE
Android Notifications

### DISCUSSION THREAD

**Discussion by j-cheung** [invertase/react-native-firebase#4993] [Q&A] [answered]
Hi,

I'm having trouble with getting notifications on Android. I can't seem to be able to get it working reliably. It only receives notifications shortly after running `react native run-android`, and then after a while it stops receiving notifications.

Any idea why this is happening? 
Any help is appreciated.
Thanks.

**Comment by mikehardy** [comment, marked as answer]
Hi there!
Quite possibly https://dontkillmyapp.com/
But you have given us no information to go on, so I won't guess further.
The issue template that we ask people to populate when they create new issues (vs discussions here) shows you the kind of information needed to actually isolate and fix issues

**Reply by j-cheung** [reply]
Hi!

Thanks for the reply, i will look into the suggestions in dontkillmyapp.com.

I didn't use the issue template because i have read through a lot of the issues and people seem to be fine with Android, so thought it is just more of a discussion. 
I will open a new issue with the relevant information then.

Thanks a lot!
